### DISCUSSION TITLE
Norway: KE Nett

### DISCUSSION THREAD

**Discussion by SVH-Powel** [JaccoR/hass-entso-e#31] [Price Modifyer Templates]
{% set s = {
    "day": 0.6676,
    "night": 0.5426,
    "cert": 0.01,
    "vat": 1.25,
    "eur": 10.36
}
%}
{% if now().isoweekday() >= 1 and now().isoweekday() <= 5 %}
  {% if now().hour >= 6 and now().hour < 22 %}
    {{(current_price * s.eur * s.vat + s.day + s.cert) | float}}
  {% else %}
    {{(current_price * s.eur * s.vat + s.night + s.cert) | float}}
  {% endif %}
{% else %}
  {{(current_price * s.eur * s.vat + s.night + s.cert) | float}}
{% endif %}

**Comment by JaccoR** [comment]
In the new release, VAT can be added as a config option. This means the template does not need the VAT anymore.

**Comment by katla50** [comment]
Made a template for Bømlo kraftnett

{% set s = {
    "day": states('input_number.nettleie') | float,
    "night": states('input_number.nettleie_natt') | float,
    "cert": states('input_number.paaslag_strom') | float,
    "vat": 1.25,
    "eur": float(states('sensor.kurs_eur'), 10)
}
%}
  {% if now().hour >= 6 and now().hour < 22 %}
    {{(current_price * s.eur * s.vat + s.day + s.cert) | float}}
  {% else %}
    {{(current_price * s.eur * s.vat + s.night + s.cert) | float}}
  {% endif %}

code for the currency stolen from stigvi:

- platform: rest
    resource: https://data.norges-bank.no/api/data/EXR/B.EUR.NOK.SP?lastNObservations=1&format=sdmx-json
    name: kurs_eur
    scan_interval: 43200
    unit_of_measurement: NOK
    value_template: "{{ value_json['data']['dataSets'][0]['series']['0:0:0:0']['observations']['0'][0] }}"
